janusgraph-dist/src/assembly/static/conf/remote.yaml

```diff
@@ -14,4 +14,4 @@
 
 hosts: [localhost]
 port: 8182
-serializer: { className: org.apache.tinkerpop.gremlin.driver.ser.GraphBinaryMessageSerializerV1, config: { serializeResultToString: true }}
```

Comment:
I've just noticed that I'm now getting a warning if I try to use this config file with our latest Docker images:

```java
gremlin> :remote connect tinkerpop.server conf/remote.yaml session
Sep 13, 2023 2:23:59 PM org.yaml.snakeyaml.internal.Logger warn
WARNING: Failed to find field for org.apache.tinkerpop.gremlin.driver.Settings.serializers
```

Do you know what could be causing this, @li-boxuan?

It however continues to work afterwards. I haven't tested however whether the serializer config is actually used or not.

Reply:
I didn't notice this warning - thanks for bringing it up. I am guessing it's caused by some dependency upgrade in TinkerPop, maybe they bumped the snakeyaml dependency and broke backward compatibility.

Reply:
// WARNING: Failed to find field for org.apache.tinkerpop.gremlin.driver.Settings.serializers

Shouldn't this be  `org.apache.tinkerpop.gremlin.server.Settings.serializers` ?

Reply:
> Shouldn't this be `org.apache.tinkerpop.gremlin.server.Settings.serializers` ?

Yeah so the warning itself doesn't quite make sense. Somehow the code believes the driver should also have this `serializers` setting, and since the driver doesn't, snakeyaml logs a warning message.